python_modules/libraries/dagster-duckdb-pandas/dagster_duckdb_pandas/duckdb_pandas_type_handler.py

```diff
@@ -128,3 +134,13 @@ def my_table_a(my_table: pd.DataFrame) -> pd.DataFrame:
             ...
 
 """
+
+
+class ConfigurableDuckDBPandasIOManager(ConfigurableDuckDBIOManager):
```

Comment:
Just to make sure, this will be the user-implemented pattern for e.g. if I make `MyTypeHandler`:

```python
class MyCustomDuckDBIOManager(ConfigurableDuckDBIOManager):
    @staticmethod
    def type_handlers() -> Sequence[DbTypeHandler]:
        return [MyTypeHandler()]
    
    # not strictly necessary, since it will be pulled from MyTypeHandler(?)
    @staticmethod
    def default_load_type() -> Optional[Type]:
        return MyType
```

Reply:
yep that's it! and yes `default_load_type` will only be required if there are multiple type handlers AND you want a default type